arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1234,6 +1234,11 @@
 				<string>This sample demonstrates how to access a map service that is secured with ArcGIS token-based authentication.</string>
 				<key>storyboardName</key>
 				<string>TokenAuthentication</string>
+				<key>dependency</key>
```

Comment:
Is this dependency in the right place? The storyboard in the previous line points to `<string>TokenAuthentication</string>`